CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Allow custom font size on action links ([PR #2162](https://github.com/alphagov/govuk_publishing_components/pull/2162))
+* Amend layout_for_public component ([PR #2160](https://github.com/alphagov/govuk_publishing_components/pull/2160))
```

Comment:
You might as well merge this into the only other commit, since it relates directly to it.